vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -60,14 +60,15 @@
 # might have to upgrade PIP itself before installing openpyxl
 - name: update PIP if necessary
   become: yes
-  pip:
+  pip:      
```

Comment:
Why not add `executable: pip3` here as well? I guess the `pip` module by itself will upgrade pip2, not pip3...